scripts/eqtl_hail_batch/launch_eqtl_spearman.py

```diff
@@ -1268,12 +1272,21 @@ def spearman_correlation(df):
     # turn into hail table and annotate with global bp and allele info
     t = hl.Table.from_pandas(adjusted_spearman_df)
     t = t.annotate(global_bp=hl.locus(t.chrom, hl.int32(t.bp)).global_position())
+    # add in vep annotation
```

Comment:
Was this change intentional? I'm asking because in the thread you said:

> For point 2, the functional annotation is there, it’s just not available for every variant

Does this change the annotation logic?

Reply:
I'm just addressing @daniaki 's point (slack message [here](https://centrepopgen.slack.com/archives/C01FBL6NWFN/p1664429696954009?thread_ts=1664149768.063269&cid=C01FBL6NWFN)) about making the subsequent rounds of the eQTL tables the same:

> Would it be simple to modify the round*/sigsnps/*.parquet outputs by adding functional_annotation...

So here I'm just adding in functional annotation for rounds 2-5

Reply:
Ah, thanks for explaining!